### ISSUE TITLE
Forecast Model Builder - feat: include training RMSE, MAE, and MAPE

### ISSUE THREAD

**Issue by sfc-gh-kmason** [Snowflake-Labs/emerging-solutions-toolbox#260]
Comparing the training set metrics to the validation set metrics can help determine if overfitting has occurred. So maybe we could include some metrics from the training set.
